openstudiocore/CMake/FindEnergyPlus.cmake

```diff
@@ -50,9 +50,9 @@ else()
   file(GLOB ENERGYPLUS_POSSIBLE_PATHS "/usr/local/[eE]nergy[pP]lus?*")
 endif()
 
-# sort possible paths in reverse order
-list(APPEND ENERGYPLUS_POSSIBLE_PATHS $ENV{ENERGYPLUSDIR})
+# sort possible paths in reverse order, favoring the ENERGYPLUSDIR environment variable
```

Comment:
The ability to use a custom E+ build might be needed on alternative platforms right?